parl/remote/worker.py

```diff
@@ -116,7 +115,8 @@ def _create_sockets(self):
             "tcp://*")
         self.reply_master_address = "{}:{}".format(self.worker_ip,
                                                    reply_master_port)
-
+        logger.set_dir(os.path.expanduser('~/.parl_data/worker/{}'.format(
```

Comment:
Why don't you put this line in the first line of __init__ function, just like what you have done in `Master.py`.

Reply:
Each worker has a unique address, which is not available at the beginning of `init`.

Reply:
I see.